src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -44,17 +48,30 @@ internal sealed class TaggerProvider : IViewTaggerProvider, ITableDataSource, II
     {
         internal readonly ITableManager ErrorTableManager;
         internal readonly ITextDocumentFactoryService TextDocumentFactoryService;
+        internal readonly IContentTypeRegistryService ContentTypeRegistryService;
+        internal readonly IFileExtensionRegistryService FileExtensionRegistryService;
+        internal readonly _DTE dte;
 
         private readonly List<SinkManager> managers = new List<SinkManager>();
         private readonly TrackerManager trackers = new TrackerManager();
 
         private readonly ISonarLintDaemon daemon;
+        private readonly ISonarLintSettings settings;
+
 
         [ImportingConstructor]
-        internal TaggerProvider([Import] ITableManagerProvider provider, [Import] ITextDocumentFactoryService textDocumentFactoryService, [Import] ISonarLintDaemon daemon)
+        internal TaggerProvider([Import] ITableManagerProvider provider,
+            [Import] ITextDocumentFactoryService textDocumentFactoryService,
```

Comment:
`[Import]` attribute is not needed but I guess you just copy/pasted what was there.